tools/new_minor_OCP_version.py

```diff
@@ -51,12 +64,18 @@ def main(args):
             latest_ocp_version,
             current_assisted_service_ocp_version))
 
-        task = create_task(args, current_assisted_service_ocp_version, latest_ocp_version)
+        ticket_id = create_task(args, current_assisted_service_ocp_version, latest_ocp_version)
+
         # task is not created in case task is available
-        if task:
-            branch = update_ai_repo_to_new_ocp_version(args, current_assisted_service_ocp_version, latest_ocp_version, task)
-            pr = open_pr(args, current_assisted_service_ocp_version, latest_ocp_version , task)
-            test_changes(branch, pr)
+        if ticket_id is not None:
+            branch, openshift_versions_json = update_ai_repo_to_new_ocp_version(args, current_assisted_service_ocp_version, latest_ocp_version, ticket_id)
```

Comment:
Add logging

Reply:
Is this comment relevant for this particular line, or the file in the general?

Reply:
Relevant for this if, add logging about the ticket you are working on